client/src/app.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+class Reviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: []
+    }
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: '/reviews',
```

Comment:
This call works, but doesn't match what is shown in the App and Service plan. I would work on getting those calls to match.. but for now it does call you server and server serves up a response!

Reply:
will be updating soon.